llvm/tools/dsymutil/CMakeLists.txt

```diff
@@ -20,6 +28,7 @@ set(LLVM_LINK_COMPONENTS
   Support
   Target
   TargetParser
+  TransformUtils
```

Comment:
Discussed offline: we should avoid the inclusion of optimization layer in MC tools.

Reply:
The pass is now enabled in our AIE Clang driver.